src/navigation/ackermann_motion_primitives.cc

```diff
@@ -111,51 +120,64 @@ vector<shared_ptr<PathRolloutBase>> AckermannSampler::GetSamples(int n) {
       auto sample = new ConstantCurvatureArc(c);
       SetMaxPathLength(sample);
       CheckObstacles(sample);
-      sample->angular_length = fabs(sample->length * c);
       samples.push_back(shared_ptr<PathRolloutBase>(sample));
     }
-  } else {
+  } else if(false){
```

Comment:
Looks like the sampler changed. If this older procedure is inferior, let's delete it.